functions-and-scope.js

```diff
@@ -70,6 +113,19 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // * Hoe zorgt ik ervoor dat wanneer ik een cijfer tegenkom die aan de conditie voldoet, ik dit ergens kan opslaan?
 // Log het antwoord in de terminal.
 
+function highestGrades(array) {
+    let highest = 0;
```

Comment:
Het is beter om als default waarde altijd het eerste cijfer van de array te pakken. Cijfers zullen in dit voorbeeld altijd hoger dan 0 zijn, maar voer je functie eens uit met een array als `[-1, -2, -4, -1]`. Dan zal er `0` uitkomen terwijl die waarde niet eens in de array voorkomt 😉